package.json

```
@@ -19,11 +19,11 @@
     "test:nonetwork": "npm run lint && npm run jest -- --testPathIgnorePatterns network.spec"
   },
   "files": [
-    "dist/*",
+    "dist/",
```

Currently, many libraries publish their src to npm, which is very useful for developers during the debugging phase. Therefore, my suggestion is that we also publish our src.

```suggestion
    "dist/",
    "src/",
```

---

I don't think we should publish src, it can confuse things like typescript and the whole point of an npm package is you publish the smallest amount of files required for your package to work.  None of our other packages publish `src`, this isn't going to be the first one

---

Forge packages currently publish `src`, actually (see https://github.com/electron/forge/pull/3284). The rationale at the time is we're shipping source maps which don't do any good if there's not also the `src` directory they reference.

I don't have a strong opinion on this either way, but I think if we don't publish `src`, we shouldn't publish source maps in `dist` either.